### PR TITLE
chapter12

### PR DESCRIPTION
**Pull request by xtrxllx**

### CODE REVIEW

**Comment by xtrxllx** [comment]
ready

File: app/assets/stylesheets/password_resets.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the PasswordResets controller here.
```
Comment (manhnt-0960) [inline app/assets/stylesheets/password_resets.scss]: delete this empty file

File: app/models/user.rb [hunk 2]
```diff
@@ -57,6 +57,18 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+<<<<<<< HEAD
```
Comment (manhnt-0960) [inline app/models/user.rb]: let's fix conflict

File: config/routes.rb [hunk 3]
```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
+  get 'password_resets/edit'
```
Comment (manhnt-0960) [inline config/routes.rb]: delete these line

**Comment by xtrxllx** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**